pkg/userdata/generate/init.go

```diff
@@ -24,21 +24,19 @@ services:
     configuration: |
       apiVersion: kubeadm.k8s.io/v1beta1
       kind: InitConfiguration
-      localAPIEndpoint:
-        bindPort: 6443
       bootstrapTokens:
       - token: '{{ .KubeadmTokens.BootstrapToken }}'
         ttl: 0s
+      localAPIEndpoint:
+        bindPort: 6443
       nodeRegistration:
-        taints: []
-        kubeletExtraArgs:
-          node-labels: ""
+        criSocket: /run/containerd/containerd.sock
```

Comment:
This was getting picked up as a docker shim something; since we don't install docker I'm not sure this was ever working(?)